metron-platform/metron-common/src/main/java/org/apache/metron/common/aggregator/Aggregators.java

```diff
@@ -18,18 +18,21 @@
 
 package org.apache.metron.common.aggregator;
 
+import org.apache.metron.common.utils.ConversionUtils;
+
 import java.util.List;
 import java.util.Map;
 import java.util.function.BinaryOperator;
 import java.util.function.Predicate;
 
 public enum Aggregators implements Aggregator {
-   MAX( (numbers, config) -> accumulate(0d, (x,y) -> Math.max(x.doubleValue(),y.doubleValue()), numbers))
-  ,MIN( (numbers, config) -> accumulate(0d, (x,y) -> Math.min(x.doubleValue(),y.doubleValue()), numbers))
-  ,SUM( (numbers, config) -> accumulate(0d, (x,y) -> x.doubleValue() + y.doubleValue(), numbers))
+   MAX( (numbers, config) -> accumulate(0d, (x,y) -> Math.max(x.doubleValue(),y.doubleValue()), numbers, config))
+  ,MIN( (numbers, config) -> accumulate(0d, (x,y) -> Math.min(x.doubleValue(),y.doubleValue()), numbers, config))
+  ,SUM( (numbers, config) -> accumulate(0d, (x,y) -> x.doubleValue() + y.doubleValue(), numbers, config))
   ,MEAN( (numbers, config) -> scale(SUM.aggregate(numbers, config), numbers, n -> true))
-  ,POSITIVE_MEAN( (numbers, config) -> scale(SUM.aggregate(numbers, config), numbers, n -> n.doubleValue() > 0))
+  ,POSITIVE_MEAN( (numbers, config) -> positiveMean(numbers, config))
   ;
+  public static String NEGATIVE_VALUES_TRUMP_CONF = "negativeValuesTrump";
```

Comment:
I noticed an issue with threat triage in-that we have no ability to cause a particular rule to trump the rest.  For instance, if you have rules 1 through 5, but if you hit rule 5, you want to make sure that the triage level is very high.  For that use-case, if you associate a negative value with a triage rule, the aggregators will return +Infinity.